kmmo/README.md

```diff
@@ -43,7 +43,7 @@ $ oc label node <node_name> intel.feature.node.kubernetes.io/dgpu-canary=true
 
 3.	Use pre-build mode to deploy the driver container.
 ```
-$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/1.0.0/kmmo/intel-dgpu.yaml   
+$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/kmmo/intel-dgpu.yaml   
```

Comment:
We decide to change to the main branch. On the development(main) branch, using the main branch URL is the right thing to do. But in the future documents which are organized according to each release, should use the release included URL. I think we should make it work in the documentation automation.